collector-bitmovin-player/src/test/java/com/bitmovin/analytics/bitmovin/player/utils/AdBreakMapperTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.bitmovin.analytics.bitmovin.player.utils
+
+import com.bitmovin.analytics.ads.AdPosition
+import com.bitmovin.player.model.advertising.AdBreak
+import com.bitmovin.player.model.advertising.AdConfiguration
+import com.bitmovin.player.model.advertising.AdTag
+import com.bitmovin.player.model.advertising.AdTagType
+import com.bitmovin.player.model.advertising.ima.ImaAdBreak
+import junit.framework.Assert
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.MockitoAnnotations
+
+class AdBreakMapperTest {
+
+    @Mock
+    private lateinit var adConfiguration: AdConfiguration
+
+    @Mock
+    private lateinit var adBreak: AdBreak
+
+    @Mock
+    private lateinit var imaAdBreak: ImaAdBreak
+
+    private var adBreakMapper = AdBreakMapper()
+
+    private val TEN_SECONDS = 10.0
+    private val ID = "id"
+    private val PRE = "pre"
+
+    @Before
+    fun setup(){
+        MockitoAnnotations.initMocks(this)
+        `when`(adConfiguration.replaceContentDuration).thenReturn(TEN_SECONDS)
+
+        `when`(adBreak.id).thenReturn(ID )
+        `when`(adBreak.scheduleTime).thenReturn(TEN_SECONDS)
+
+        `when`(imaAdBreak.id).thenReturn(ID )
+        `when`(imaAdBreak.tag).thenReturn(AdTag("", AdTagType.VMAP))
+        `when`(imaAdBreak.position).thenReturn(PRE)
+    }
+
+    @Test
+    fun FromPlayerAdConfigurationWithAdConfigurationShouldSetValuesOfAdConfiguration(){
+        val collectorAdBreak = adBreakMapper.FromPlayerAdConfiguration(adConfiguration)
+
+        assertThat(collectorAdBreak.replaceContentDuration).isEqualTo(TEN_SECONDS)
+    }
+
+    @Test
+    fun FromPlayerAdConfigurationWithAdConfigurationShouldNotSetValuesOfAdBreak(){
+        val collectorAdBreak = adBreakMapper.FromPlayerAdConfiguration(adConfiguration)
+
+        assertThat(collectorAdBreak.scheduleTime).isEqualTo(TEN_SECONDS)
```

Comment:
Is this really in seconds? In the sample shouldn't this be a millisecond timestamp?